### ISSUE TITLE
Quito is NOT in Antarctica!

### ISSUE THREAD

**Issue by ah132f** [jpatokal/openflights#437]
I've just noticed that somebody has moved Quito (IATA code UIO) - it's now showing as in the middle of Antarctica!

Looking further, it seems the latitude and longitude have been swapped - can somebody fix this, please!

**Comment by wshaw17** [comment]
ditto--i'm also seeing this issue with Quito UIO.

**Comment by jpatokal** [comment]
Sorry, looks like this got messed up last week when moving to the new Quito airport.  Should be fixed now, thanks for reporting!

http://openflights.org/airport/UIO

**Comment by wshaw17** [comment]
Thank you! 

```
On Sunday, December 20, 2015 6:49 PM, Jani Patokallio <notifications@github.com> wrote:
```

 Closed #437.—
